extensions/amp-list/amp-list.md

```diff
@@ -578,3 +580,5 @@ The AMP for Email spec disallows the use of the following attributes on the AMP
 ## Validation
 
 See [amp-list rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/validator-amp-list.protoascii) in the AMP validator specification.
+
+[/filter] <!-- formats="websites, stories" -->
```

Comment:
nit: Instead of adding a closing filter tag, let's remove the format opening tag prior to the Validation section. All eligible formats should have a link to the validator spec, no?